src/Integration.Vsix/InfoBar/InfoBarManager.cs

```diff
@@ -87,6 +87,46 @@ public IInfoBar AttachInfoBar(Guid toolWindowGuid, string message, SonarLintImag
             return AttachInfoBarImpl(toolWindowGuid, message, imageMoniker);
         }
 
+        public IInfoBar AttachInfoBarWithButtonMainWindow(string message, string buttonText, SonarLintImageMoniker imageMoniker)
```

Comment:
Almost identical to the already existing methods expect they dont need a guid and they call AttachInfoBarMainWindowImpl instead of AttachInfoBarmpl